### PR TITLE
Feat: Implemented authentication using Lucia Auth, Postgress and next Cookies

### PR DESCRIPTION
**Pull request by AntGa**

### CODE REVIEW

File: src/app/dashboard/page.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { getUser } from '@/lib/lucia'
+import { redirect } from 'next/navigation'
+import React from 'react'
+
+const page = async () => {
+  const user = await getUser()
+  if (!user) {
+    redirect('/login')
+  }
+
```
Comment (tulza) [inline src/app/dashboard/page.tsx:11]: should probably have a ui loading state

File: tailwind.config.ts [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
   theme: {
     extend: {
       colors: {
-        background: "var(--background)",
-        foreground: "var(--foreground)",
+        'background-dark': '#282A36',
+        'background-darker': '#20222C',
+        'primary-accent': '#9D7CCF',
+        'secondary-blue': '#6272A4',
+        'highlight-lavender': '#9D7CCF',
```
Comment (tulza) [inline tailwind.config.ts]: make this a var color in css then put those in tailwind

File: src/app/login/SignUpForm.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            <p className="text-red-500">{errors.email.message}</p>
+          )}
+        </div>
+        <div className="mb-2">
+          <input
+            type="password"
+            id="password"
+            {...register('password')}
+            placeholder="Password"
+            className="w-full overflow-hidden rounded-xl border border-transparent bg-background-darker px-5 py-2 text-sm text-white placeholder-secondary-blue focus:border-white focus:outline-none"
```
Comment (tulza) [inline src/app/login/SignUpForm.tsx]: make these input a component

**Review by tulza — APPROVED** [review]
lgtm

File: tailwind.config.ts (hunk 2 above)
Comment (AntGa) [reply tailwind.config.ts]: will do now 😄

File: src/app/login/SignUpForm.tsx (hunk 3 above)
Comment (AntGa) [reply src/app/login/SignUpForm.tsx]: yes sir

**Pull request merged**